content/applications/general/export_import_data.rst

```diff
@@ -350,62 +417,64 @@ It will produce the following CSV file:
    person_3,Eric,False,company_2
    person_4,Ramsy,False,company_3
 
-As you can see in this file, Fabien and Laurence are working for the Bigees company (company_1) and
+As can be seen in this file, Fabien and Laurence are working for the Bigees company (company_1) and
 Eric is working for the Organi company. The relation between persons and companies is done using the
-External ID of the companies. We had to prefix the "External ID" by the name of the table to avoid a
+`External ID` of the companies. The `External ID` is prefixed by the name of the table to avoid a
 conflict of ID between persons and companies (person_1 and company_1 who shared the same ID 1 in the
 original database).
 
 The two files produced are ready to be imported in Odoo without any modifications. After having
-imported these two CSV files, you will have 4 contacts and 3 companies. (the firsts two contacts are
-linked to the first company). You must first import the companies and then the persons.
+imported these two :abbr:`CSV (Comma-separated Values)` files, there are four contacts and three
+companies (the first two contacts are linked to the first company). Keep in mind to first import
+the companies and then the persons.
 
 How to adapt an import template
 ===============================
 
-Import templates are provided in the import tool of the most common data to
-import (contacts, products, bank statements, etc.).
-You can open them with any spreadsheets software (Microsoft Office,
+Import templates are provided in the import tool of the most common data to import (contacts,
+products, bank statements, etc.). Open them with any spreadsheet software (Microsoft Office,
 OpenOffice, Google Drive, etc.).
 
-How to customize the file
-=========================
+How to customize file
+=====================
 
-* Remove columns you don't need. We advise to not remove the *ID* one (see
-  why here below).
+* Remove columns that are not needed. Odoo advises not removing the :guilabel:`External ID` (ID)
+  column (see why below).
 * Set a unique ID to every single record by dragging down the ID sequencing.
 
   .. image:: export_import_data/dragdown.gif
      :align: center
+     :alt: Animation of a spreadsheet application automatically populating a column in the ID.
 
-* When you add a new column, Odoo might not be able to map it automatically if its
-  label doesn't fit any field of the system.
+* When adding a new column, Odoo might not map it automatically if its label does not match any
+  field in the system.
   If so, find the corresponding field using the search.
```

Comment:
Early line break or missing blank line (depending on how you want this to format)